spk/python310/src/requirements-crossenv.txt

```diff
@@ -8,98 +8,84 @@
 # Require environment variables
 #  PYO3_CROSS_LIB_DIR=$(STAGING_INSTALL_PREFIX)/lib/
 #  PYO3_CROSS_INCLUDE_DIR=$(STAGING_INSTALL_PREFIX)/include/
-# bcrypt==4.0.1
-# cryptography==38.0.4
+bcrypt==4.0.1
+cryptography==38.0.4
 
 # Requirement for cryptography
-# cffi==1.15.1
+cffi==1.15.1
 
 # commonly used cross-compiled wheels
 # All tested and can be enabled but may conflict
 # with local python apps requirements
 
-# greenlet==1.1.3
-# immutables==0.19
-# MarkupSafe==2.1.1
-# msgpack-python==0.5.6
-# netifaces==0.11.0
-# regex==2022.9.13
-# SQLAlchemy==1.4.41
-# zope.interface==5.4.0
+greenlet==2.0.1
+immutables==0.19
+MarkupSafe==2.1.1
+msgpack-python==0.5.6
+netifaces==0.11.0
+regex==2022.10.31
+SQLAlchemy==1.4.45
+zope.interface==5.5.2
 
 # [gevent]
 # Depends: libev, c-ares
 # Require environment variables
 #  GEVENTSETUP_EMBED_CARES=FALSE
 #  GEVENTSETUP_EMBED_LIBEV=FALSE
-# gevent==21.12.0
+gevent==22.10.2
 
 # [lxml]
 # Depends: libxml2, libxslt
-# lxml==4.9.1
+lxml==4.9.2
 
 # [mysqlclient]
 # Depends: mysql-connector-c, mariadb-connector-c
 # Require environment variables
 #  MYSQLCLIENT_CFLAGS
 #  MYSQLCLIENT_LDFLAGS
-# mysqlclient==2.1.1
-
-# [numpy]
-#   - armv5 (88f6281-6.1) Last known working version
-# numpy==1.21.6
-
-# [numpy]
-#   - armv7L (hi3535-6.1) Last known working version
-# numpy==1.22.4
-
-# [numpy]
-#   - Require Cython in cross/python310 crossenv
-#   - DSM6 require WHEELS_CFLAGS=-std=c++0x
-#   - aarch64-6.1 require workaround WHEELS_CFLAGS=-O0
-#     ref: https://github.com/numpy/numpy/issues/13622
-# numpy==1.23.3
+mysqlclient==2.1.1
 
 # [Pillow]
 # Require --global-options arguments
 #  WHEELS_BUILD_ARGS = [Pillow] build_ext --disable-platform-guessing
-# Pillow==9.2.0
+Pillow==9.3.0
 
 # [psutil]
 # Last known version to build on armv5/ppc
-# psutil==5.9.1
+psutil==5.9.1
+#psutil==5.9.4
```

Comment:
psutil==5.9.4 builds for qoriq (ppc), version 5.9.2 had a bug that was fixed with https://github.com/giampaolo/psutil/pull/2156

Reply:
ah! thnx, i was looking for exactly that!